src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/NuGetServices.cs

```diff
@@ -14,23 +14,27 @@ public static class NuGetServices
         private const string SolutionManagerServiceName = "Microsoft.VisualStudio.NuGet.SolutionManagerService";
         private const string SolutionManagerServiceVersion = "1.0.0";
         private const string SourceProviderServiceName = "Microsoft.VisualStudio.NuGet.SourceProviderService";
-        private const string SourceProviderServiceVersion = "1.0.0";
+        private const string SourceProviderServiceVersion_1_0_1 = "1.0.1";
```

Comment:
This breaks compat between versions right?  This version of NuGet on the server can no longer work with old clients requesting service version 1.0.0, and this version of NuGet on the client won't fall back to service version 1.0.0 on the server.

Reply:
yes, given that codespaces is early preview, we agreed with this approach for now.
will list this feedback in https://github.com/NuGet/Home/issues/10393 so that i confirm with @sbanni when he returns from vacation.